pkg/cmd/attestation/verify/verify.go

```diff
@@ -150,6 +150,7 @@ func NewVerifyCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Command
 	// general flags
 	verifyCmd.Flags().StringVarP(&opts.BundlePath, "bundle", "b", "", "Path to bundle on disk, either a single bundle in a JSON file or a JSON lines file with multiple bundles")
 	cmdutil.DisableAuthCheckFlag(verifyCmd.Flags().Lookup("bundle"))
+	verifyCmd.Flags().BoolVarP(&opts.UseBundleFromRegistry, "bundle-from-registry", "", false, "Use the bundle from the OCI registry")
```

Comment:
Just making a note here that before we merge this I'd like to have a quick think about the name of this flag 🤔.

Reply:
(I promise to resolve this by end of tomorrow)

Reply:
Is this only for OCI images, or any artifact? Maybe that could be clarified.

Reply:
Only for OCI images. maybe I should add logic to throw error message if provide non OCI images and with bundle-from-registry falg?